do not attempt to attach puppet class at host creation if puppet environment is not imported into given organization

tests.foreman.cli.test_host.HostCreateTestCase.test_positive_create_with_puppet_class_id

Stacktrace
```
self = <tests.foreman.cli.test_host.HostCreateTestCase testMethod=test_positive_create_with_puppet_class_id>

    @tier1
    def test_positive_create_with_puppet_class_id(self):
        """Check if host can be created with puppet class id
    
            :id: 6bb1bbdc-23fd-4493-9283-fbb70d72b2eb
    
            :expectedresults: Host is created and has puppet class assigned
    
            :CaseImportance: Critical
            """
        host = make_fake_host({
            'puppet-class-ids': self.puppet_class['id'],
            'environment-id': self.puppet_env['id'],
        })
        host_classes = Host.puppetclasses({'host-id': host['id']})
        self.assertIn(
            self.puppet_class['id'],
>           [puppet['id'] for puppet in host_classes]
        )
E       AssertionError: '33' not found in []

tests/foreman/cli/test_host.py:572: AssertionError
```

See https://bugzilla.redhat.com/show_bug.cgi?id=1595161#c7 and below

--- a/tests/foreman/cli/test_host.py
+++ b/tests/foreman/cli/test_host.py
@@ -55,6 +55,7 @@ from robottelo.cli.hostcollection import HostCollection
 from robottelo.cli.lifecycleenvironment import LifecycleEnvironment
 from robottelo.cli.medium import Medium
 from robottelo.cli.operatingsys import OperatingSys
+from robottelo.cli.org import Org
 from robottelo.cli.package import Package
 from robottelo.cli.proxy import Proxy
 from robottelo.cli.puppet import Puppet
@@ -146,6 +147,12 @@ class HostCreateTestCase(CLITestCase):
         cls.puppet_class = Puppet.info({
             'name': puppet_modules[0]['name'],
             'environment': cls.puppet_env['name'],
+        })
+        # adding org to a puppet env
+        Org.set_parameter({
+            'name': 'Environment',
+            'value': cls.puppet_env["name"],
+            'organization': cls.new_org["name"],
         })
 
     def setUp(self):

